components/dropdown/test/dropdown-content.test.js

```diff
@@ -412,21 +394,6 @@ describe('d2l-dropdown', () => {
 			expect(content.opened).to.be.true;
 		});
 
-		it('should focus on container when opened and no focusable light descendant exists', async() => {
```

Comment:
This focus logic is handled by the `PopoverMixin`, and there is an existing unit test for it.